### PR TITLE
[actuators] Add UAVCAN actuators for chibios

### PR DESCRIPTION
**Pull request by fvantienen**
This currently works, but I think we should split the UAVCAN from the actuators. That way we can also implement sensors and other UAVCAN equipment.

### CODE REVIEW

**Comment by gautierhattenberger** [comment]
I totally agree that we should split this in two, and even hide the ChibiOS implementation behind a proper can peripheral interface. The current one is not generic at all at the moment.
So basically, I don't know what to do with this PR... Although it is a nice addition, It should be completely rewritten, but this may take a long time.

File: sw/airborne/arch/chibios/subsystems/actuators/actuators_uavcan.c [hunk 1]
```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (C) 2013 AggieAir, A Remote Sensing Unmanned Aerial System for Scientific Applications
+ * Utah State University, http://aggieair.usu.edu/
+ *
+ * Michal Podhradsky (michal.podhradsky@aggiemail.usu.edu)
```
Comment (podhrmic) [inline sw/airborne/arch/chibios/subsystems/actuators/actuators_uavcan.c]: I suspect the copyright can be updated as we were not involved into making this file:-)

Comment (fvantienen) [reply sw/airborne/arch/chibios/subsystems/actuators/actuators_uavcan.c]: Will change that ^^

**Comment by fvantienen** [comment]
Depending on the time I have now I will decide if I want to merge this somewhat like this or in a better way.

**Comment by gautierhattenberger** [comment]
@fvantienen @OpenUAS it seems there are plans to make a proper CAN + UAVCAN driver, should we keep this ad-hoc implementation ?

**Comment by gautierhattenberger** [comment]
if you can resolve the conflict on TUdelft conf, I'm okay to merge this until there is a proper support of CAN and UAVCAN

**Review by gautierhattenberger — APPROVED** [review]
thank you for splitting uavcan driver from actuator module

**Comment by gautierhattenberger** [comment]
CI is failing for unrelated reasons, so it should be fine since it doesn't really impact other things

**Pull request merged**